### ISSUE TITLE
Config file 'level' almost never used

### ISSUE THREAD

**Issue by EdwinHuijsing** [BradCrumb/atom-linter-phpstan#14]
By default the level value is set to `0`  in atom , due to this value the level is always passed to the command-line.

In main.js at line 135
```js
  if (levels.indexOf(this.level) != -1) {
    parameters.push('-l', this.level);
  }
```
As you can see in the above snippet, to prevent the level to be passed to the command-line I have to fill a value of `-1` in atom. This is also not correct, because don't always have a config file to use.

The check needs to be more like this:
```js
  if (!confFile) {
    if (levels.indexOf(this.level) != -1) {
      parameters.push('-l', this.level);
    }
  }
```

Want to check-it for your self.

Step 1.
In atom set the level to `0` and in the config file set the value to `8`.

Step 2.
Run phpstan from the command-line.

Step 3.
Validate the file that have error in atom.

**Comment by mallardduck** [comment]
@EdwinHuijsing - This should be fixed in version 2.2.0 now. The way I implemented it was by adding a new setting for if the level of the linter should override the level of the config file. This setting has a default of false.

This way moving forward if a user provides a config file then the Level config value will not be used. However if they provide a config file and want the level to override the config (for w/e reason) then if they enable this setting the Level will be set along with the config.

**Comment by mallardduck** [comment]
Assuming you are still using Atom and this linter I would ask you update the plugin. If you are still seeing issues once you are on v2.2.0 then we can reopen this issue to review.
